af/fastpath/fastpath/s3feeder.py

```diff
@@ -66,23 +66,25 @@ def load_multiple(fn: str) -> Generator[MsmtTup, None, None]:
                 if m.name.endswith(".json"):
                     for line in k:
                         msm = ujson.loads(line)
-                        msmt_uid = trivial_id(msm)
+                        msmt_uid = trivial_id(line, msm)
+                        msm["measurement_uid"] = msmt_uid
```

Comment:
Ok cool, so we are adding the `measurement_uid` to the output JSON entries. 

We should remember to update ooni/spec to reflect this improvement to the output data format.

Reply:
This was based on a discussion on Slack. Now legacy msmts have the UID in them.